bhclub.html

```diff
@@ -44,6 +44,45 @@ <h1>BrickHack</h1>
         </section>
         <section id="about">
             <h1>About</h1>
+            <p>A hackathon is a “hacking marathon” in which teams or individuals have a specified time frame to create something from nothing. These events foster creative problem solving, teamwork, and innovative thinking within the hackathon community.</p>
```

Comment:
I would consider the first sentence to be a definition and the second one to be a comment on the nature of hackathons. 

I agree that the second sentence is a bit off, perhaps a bit too formal. That goes for a lot of the content written right now. It would be helpful for future PRs if comments about the other content could be made in the google doc I sent out.

Reply:
> I would consider the first sentence to be a definition and the second one to be a comment on the nature of hackathons.

Alright, that makes sense. Could we remove the clause "within the hackathon community" then and leave the rest as-is? It's kinda implied that hackathons have an effect on the hackathon community.

> It would be helpful for future PRs if comments about the other content could be made in the google doc I sent out.

You're right, I should put comments in there instead. I'll give that a more thorough review tomorrow.

Reply:
Changing it to "within the hackers" because something sounds off if I just cut it off.

Reply:
Changed my mind again and replacing it with "skills"